setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.3.1.dev0
+current_version = 0.0.1.dev0
```

Comment:
Is there a reason we changed the version here, and should there be a corresponding change in `setup.py`? I tested this out locally and it seems like bumpversion complains if this version doesn't match the version in `setup.py`.

Reply:
Good catch, I'll revert it there as well.